libs/nest/problem/src/lib/test/setup.ts

```diff
@@ -0,0 +1,56 @@
+import { Controller, Get, Module } from '@nestjs/common'
+import request, { Test } from 'supertest'
+import { testServer } from '@island.is/infra-nest-server'
+import { GraphQLModule, Query, Resolver } from '@nestjs/graphql'
+import { Logger, LOGGER_PROVIDER } from '@island.is/logging'
+import { ProblemModule } from '../problem.module'
+
+export type CreateRequest = (mode?: 'rest' | 'graphql') => Test
```

Comment:
I noticed the `'graphql'` string is hardcoded in serveral places. Should it be an enum?